plugins/source/aws/resources/services/cloudformation/stack_instance_resource_drifts.go

```diff
@@ -0,0 +1,72 @@
+package cloudformation
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/resources/services/cloudformation/models"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func stackInstanceResourceDrifts() *schema.Table {
+	table_name := "aws_cloudformation_stack_instance_resource_drifts"
+	return &schema.Table{
+		Name: table_name,
+		Description: `https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_StackSetOperationResultSummary.html.
+The 'request_account_id' and 'request_region' columns are added to show the account and region of where the request was made from.`,
+		Resolver:  fetchStackInstanceResourceDrifts,
+		Transform: transformers.TransformWithStruct(&types.StackInstanceResourceDriftsSummary{}),
```

Comment:
I left out PKs from the table.

Reply:
I think you can use all of the inputs (minus `CallAs`) as pks + `logical_resource_id` or `physical_resource_id`

Reply:
I think logical and physical both need to be used in that case, but it gets [complicated](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_StackInstanceResourceDriftsSummary.html):

```go
	// Context information that enables CloudFormation to uniquely identify a
	// resource. CloudFormation uses context key-value pairs in cases where a
	// resource's logical and physical IDs aren't enough to uniquely identify that
	// resource. Each context key-value pair specifies a unique resource that contains
	// the targeted resource.
	PhysicalResourceIdContext []PhysicalResourceIdContextKeyValuePair
```

Reply:
But we don't need to uniquely identify a resource in a region/account, we just need to uniquely identify a resource within a specific stack